### ISSUE TITLE
CIS 1.1.16 Ensure noexec option set on /dev/shm partition

### ISSUE THREAD

**Issue by jackfrancis** [Azure/aks-engine#967]
Here's the chef mount syntax for this one:

```
mount 'Mount /dev/shm' do
  pass     0
  fstype   'tmpfs'
  device   'tmpfs'
  mount_point '/dev/shm'
  options 'rw,nosuid,nodev,noexec'
  action   :enable
end
```

**Comment by hieumoscow** [comment]
I still see noexec option is not set.
AKSUbuntu-1804containerd-2021.12.07
Will this be included in future updates?
@jackfrancis 

```bash
root@aks-agentpool-19894834-vmss000000:/# findmnt -n /dev/shm | grep -v noexec
/dev/shm tmpfs  tmpfs  rw,nosuid,nodev
```

**Comment by mboersma** [comment]
@hieumoscow we enumerated all the CIS recommendations, but didn't necessarily implement all of them. The `noexec` flag for `/dev/shm` wasn't implemented in AKS Engine, for example, we just eventually closed this issue with no action.

Judging by the name of your agentpool, you're working with an AKS cluster. [AKS is a service](https://github.com/Azure/aks-engine/blob/master/docs/faq.md#whats-the-difference-between-aks-and-aks-engine) that really has no relation to the AKS Engine project at this point, so I would suggest you pursue this with their support team.
